english/lighter-use/lighter-use.md

```diff
@@ -0,0 +1,11 @@
+The marshmallow man explains that he thinks the lighter will 
+melt the walls of the prison so you can both escape. He says that 
+the lighter will slowly melt the marshmallow, but you must not put it
+too close to the wall. If the flame touches it, the wall will 
+burst into flames and you will both be consumed in the inferno.
+
+You:
+
+[Use the lighter to melt the wall](melt-wall/melt-wall.md)
+[Tell him he's nuts](nuts/nuts.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.